mysite/models.py

```diff
@@ -1,5 +1,62 @@
 from django.db import models
```

Comment:
Also, let's have separate applications for Product and Order

Reply:
done